src/Render/utilities.jl

```diff
@@ -48,6 +48,72 @@ name(mod, q::QualifiedSymbol) = q.sym
 
 name(mod, aside::Aside) = symbol("")
 
+# Get the final name of an documented object such as ``Function`` or ``Method``.
+nameof(m::Module, obj) = name(m, obj)
+
+function nameof(m::Module, obj::Aside)
+    linenumber, path = Cache.getmeta(m, obj)[:textsource]
+    return symbol("aside_$(first(splitext(basename(path))))_L$(linenumber)")
+end
+
+function nameof(m::Module, obj::Function)
+    meta = Cache.getmeta(m, obj)
+    meta[:category] == :function ? obj.env.name : symbol(string("@", meta[:signature].args[1]))
+end
```

Comment:
Combine with the ``name`` method above. Adapting ``showobj`` so that the ``@`` is only displayed once.